util/types.go

```diff
@@ -1,8 +1,9 @@
 package util
 
 const (
-	// DefaultRuntime is the default command to use to run the container.
-	DefaultRuntime = "runc"
+	// DefaultRuntime should no longer be used.
+	// Default runtime should come from containers.conf
+	DefaultRuntime = "crun"
```

Comment:
I think this should be "runc" for now. It's safe for Fedora but who knows in which packages this ends up.

Reply:
I actually think we should remove this altogether or maybe return "".  This field should never be used.  Basically it would only ever be used if containers/common had a serious issue, where their was a coding mistake in the code, which has already gone through  multiple layers of testing.

Reply:
But to get it through, I changed it back. Also fixed the message if containers.conf is screwed up to Warnf level.